Makefile

```diff
@@ -23,6 +23,8 @@ $(DIR__CI)/patched:
 ifneq (_,_$(findstring all,$P))
 	cd $(DIR__OPENWRT)/feeds/packages; patch -p1 < $(DIR__CI)/0001-glib2-make-libiconv-dependent-on-ICONV_FULL-variable.patch; \
 	patch -p1 < $(DIR__CI)/0001-node-host-turn-off-verbose.patch; \
```

Comment:
This patch can now be removed as it has been verified that build can progress without this patch